pkg/products/monitoring/dashboards/criticalSLOAlerts.go

```diff
@@ -225,6 +226,7 @@ const MonitoringGrafanaDBCriticalSLOAlertsJSON = `{
 			"targets": [{
 				"expr": "sum(ALERTS{severity='critical', alertstate='firing'}) or vector(0)",
 				"format": "time_series",
+				"instant": false,
```

Comment:
Added in these instant fields even for places where they're not being used, I think it adds a bit more readability to the already busy Json, but I can take them out if we don't want them here.